src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -287,13 +233,13 @@ def _get_scrubbed_secrets(self):
         return {key: val for key, val in self.secrets.items() if self._is_scrubbed_value(val)}
 
 
-class _StrongTypeConnection(_Connection):
+class _StrongTypeConnection(_CoreStrongTypeConnection, _Connection):
```

Comment:
entities.OpenAIConnection can be passed to place where accepts a core.OpenAIConnection, right?

Reply:
Yes.